### DISCUSSION TITLE
Use of branches for editing page?

### DISCUSSION THREAD

**Discussion by pjkreutzer** [swinnoproject/website#14] [Ideas]
Should we require changes made by org members to be made through branching and filing a pull request, or allow direct changes onto main? @MatJohaDH

**Comment by mathjoha** [comment]
I prefer the branch->merge approach because then we can more easily deal with individual cases, inevitable errors etc.

Branches are cheap!

The only argument against this is that we need to explain how branches work -- at a very basic level.

@pjkreutzer

**Reply by pjkreutzer** [reply]
Ok. I think we also need to talk about who merges and at what frequency. Can do it every now and then if it is a quick merge, but do not want to be responsible for it.

**Reply by pjkreutzer** [reply]
I'll add the documentation to the readme
